rbac/migration_tool/migrate.py

```diff
@@ -211,3 +212,19 @@ def migrate_data(exclude_apps: list = [], orgs: list = [], write_relationships:
         count += 1
         logger.info(f"Finished migrating data for tenant: {tenant.org_id}. {count} of {total} tenants completed")
     logger.info("Finished migrating data for all tenants")
+
+
+def relationships_for_new_user(user_id, org_admin, tenant):
+    """Create relationships for new user."""
+    relationships = []
+    relationships.append(create_relationship("tenant", str(tenant.org_id), "user", user_id, "member"))
+    group_default = tenant.group_set.filter(platform_default=True).first()
+    if not group_default:  # Means it is not custom platform_default
+        group_default = DEFAULT_GROUP
+    relationships.append(create_relationship("group", str(group_default.uuid), "user", user_id, "member"))
+    if org_admin:
+        admin_default = tenant.group_set.filter(admin_default=True).first()
+        if not admin_default:
+            admin_default = DEFAULT_ADMIN_GROUP
+        relationships.append(create_relationship("group", str(admin_default.uuid), "user", user_id, "member"))
```

Comment:
I guess namespace is missing, but maybe also there are some methods in dual write group handler exists. or maybe also this https://github.com/RedHatInsights/insights-rbac/blob/c169e44d6caa2e72b096024387a1b72f4e025abd/rbac/management/group/model.py#L56.

Reply:
Yea, I removed these and reusing Alec's bootstrapping tenants